api/contrib/envoy/extensions/network/connection_balance/dlb/v3alpha/dlb.proto

```diff
@@ -34,4 +34,10 @@ message Dlb {
   // Maximum number of retries when sending to DLB device fails.
   // No retry as default.
   uint32 max_retries = 2;
+
+  // The fallback policy if any error occurs:
+  // The `true` means fall back to Exact Connection Balance.
+  // The `false` means fall back to Nop Connection Balance.
+  // The default value is false.
+  bool is_exact_fallback = 3;
```

Comment:
ISTM that putting an enum here might allow greater flexibility and extensiveness

Reply:
Should we have an option to not fallback?

Reply:
Good idea, that seems reasonable (and more easily doable) with an enum.

Reply:
Done.